### PR TITLE
Jhia 46 masonry list

### PR DESCRIPTION
**Pull request by Joshw8**
Review after https://github.com/JumpTwentyFour/wandersnap-app/pull/24

### CODE REVIEW

File: src/components/listings/MasonryListItem.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+interface MasonryListItemProps {
+  image: ImageSourcePropType
+  containerWidth: number
+}
+
+function MasonryListItem(props: MasonryListItemProps) {
+  const { image, containerWidth } = props
+
+  const blurhash =
```
Comment (danielj24) [inline src/components/listings/MasonryListItem.tsx:13]: Can we make this a const outside the component please?

```
const BLUR_FALLBACK_HASH = '..'

function MasonryListItem(props: MasonryListItemProps) {
```

File: src/components/listings/MasonryListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    '|rF?hV%2WCj[ayj[a|j[az_NaeWBj@ayfRayfQfQM{M|azj[azf6fQfQfQIpWXofj[ayj[j[fQayWCoeoeaya}j[ayfQa{oLj?j[WVj[ayayj[fQoff7azayj[ayj[j[ayofayayayj[fQj[ayayj[ayfjj[j[ayjuayj['
+
+  const [imgHeight, setImageHeight] = useState<number>(190)
```
Comment (danielj24) [inline src/components/listings/MasonryListItem.tsx:16]: const this as well please
`const IMG_HEIGHT_DEFAULT = 190`

File: src/components/listings/MasonryListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  function handleImageHeight(source: {
+    url: string
+    width: number
```
Comment (danielj24) [inline src/components/listings/MasonryListItem.tsx:20]: do we need these other parameters?

File: src/components/listings/MasonryListing.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { View, Dimensions, ImageSourcePropType } from 'react-native'
+
```
Comment (danielj24) [inline src/components/listings/MasonryListing.tsx:3]: remove new line

File: src/components/listings/MasonryListing.tsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+import MasonryListItem from '@/components/listings/MasonryListItem'
+
+interface MasonryListingProps {
+  images: ImageSourcePropType[]
+}
+
+function MasonryListing(props: MasonryListingProps) {
+  const { width } = Dimensions.get('window')
```
Comment (danielj24) [inline src/components/listings/MasonryListing.tsx:11]: should this be using the window dimensions or should we ref the root View in this file and reference that for it's width instead?

File: src/components/listings/MasonryListItem.tsx (hunk 1 above)
Comment (Joshw8) [reply src/components/listings/MasonryListItem.tsx:13]: 👍

File: src/components/listings/MasonryListItem.tsx (hunk 2 above)
Comment (Joshw8) [reply src/components/listings/MasonryListItem.tsx:16]: 👍

File: src/components/listings/MasonryListItem.tsx (hunk 3 above)
Comment (Joshw8) [reply src/components/listings/MasonryListItem.tsx:20]: 👍

**Pull request merged**